src/internal/command/listen_interruption.go

```diff
@@ -0,0 +1,18 @@
+package command
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func ListenInterruption(quit chan struct{}) {
```

Comment:
@nicobarra  maybe this implementation should be in other place because we need implement this in all projects (sync and jobs)

Reply:
Maybe if we have  a `main` module that imports all of the 3 modules and executes them, we could have the implementation there. Like any other implementation that can be useful for all the modules.

So @cagodoy  @mtavano, should we create a `main` module containing and managing Nodes, Jobs and Synchronizer-V2?